examples/statistics/customized_violin_demo.py

```diff
@@ -18,87 +18,74 @@
 import numpy as np
 
 
-# functions to calculate percentiles and adjacent values
-def percentile(vals, p):
-    N = len(vals)
-    n = p*(N+1)
-    k = int(n)
-    d = n-k
-    if k <= 0:
-        return vals[0]
-    if k >= N:
-        return vals[N-1]
-    return vals[k-1] + d*(vals[k] - vals[k-1])
-
-
 def adjacent_values(vals):
-    q1 = percentile(vals, 0.25)
-    q3 = percentile(vals, 0.75)
-    iqr = q3 - q1  # inter-quartile range
-
+    q1, q3 = np.percentile(vals, [25, 75])
+    # inter-quartile range iqr
+    iqr = q3 - q1
```

Comment:
What about removing this comment and instead changing to clearer variable names:
`interquartile_range = q3 - q1`